apkid/rules/apk/packers.yara

```diff
@@ -604,3 +604,23 @@ rule secenh : packer
     and 1 of ($a*)
     and 1 of ($b*)
 }
+
+// There can also be rules for dex files because of NRV compression but I can't find compressed classesN.dex
+// I guess it is optional(?).
+rule legu : packer
```

Comment:
Can you specify the entire name in rule and description names? e.g. tencent_legu and Tencent Legu.
Also the word `packer` will be redundant, right? This will be shown under the `packer` rules....

Reply:
The latter packer word will be redundant. E.g. ` packer -> Tencent's Legu Packer`